frontend/src/app/general/metadata/version/version.component.html

```diff
@@ -7,21 +7,23 @@
   <a
     href="/docs/release-notes/"
     target="_blank"
-    (click)="versionService.clickedOnVersionNotes()"
+    (click)="metadataService.clickedOnVersionNotes()"
   >
     <div
       class="version"
```

Comment:
Here, at least this rule, can be adjusted to tailwind. Especially as we not often touch this component it might make sense to apply this small change in this PR.

Reply:
Done and tested.